extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -1065,9 +1066,9 @@
 
 	};
 	window.ThemeDesigner = ThemeDesigner;
-})(window);
 
-$(function () {
-	'use strict';
-	window.ThemeDesigner.init();
+	$(function () {
+		'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)